content/education/manage-coursework-with-github-classroom/get-started-with-github-classroom/index.md

```diff
@@ -8,6 +8,9 @@ redirect_from:
   - /education/manage-coursework-with-github-classroom/basics-of-setting-up-github-classroom
   - /education/manage-coursework-with-github-classroom/get-started-with-github-classroom/basics-of-setting-up-github-classroom
 children:
+  - /about-github-classroom
+  - /basics-of-setting-up-github-classroom
   - /glossary
 ---
 
+i
```

Comment:
```suggestion
```

I think this might be a typo that snuck in?

Reply:
yeah it is, thanks😅